Shopify/hydrogen
# [DO NOT MERGE] Example hydrogen with cloudflare pages

This is an example of an Hydrogen setup for cloudflare pages (which should take advantage of workers and is simpler)
Followed @jplhomer's advice and used https://github.com/jacob-ebey/remix-cf-pages-template-hmr

WHY are these changes introduced?
Reports of people having issues deploying to cloudflare. Our guide is too simple, it actually takes a while to set this up

WHAT is this pull request doing?
This PR is mainly to get feedback.

How to test?
Follow the readme inside examples/cloudflare-pages
Check that yarn install and yarn dev works
If possible, create a cloudflare account and make a deploy by linking a repo
